packages/components/src/pagination-control/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { times } from 'lodash';
+import './style.scss';
+
+interface Props {
+	onChange: ( page: number ) => void;
+	activePageIndex: number;
+	numberOfPages: number;
+	classNames?: string | string[];
+}
+
+const PaginationControl: React.FunctionComponent< Props > = ( props ) => {
+	const { activePageIndex, numberOfPages, onChange, classNames, children } = props;
```

Comment:
We can destructure `props` in the signature.

```suggestion
const PaginationControl: React.FunctionComponent< Props > = ( { activePageIndex, numberOfPages, onChange, classNames, children } ) => {
```